### DISCUSSION TITLE
MC (Midnight Commander) on Ventura 13.5 Apple M1 - invalid filename sorting in pannels

### DISCUSSION THREAD

**Discussion by mjosifek** [Homebrew/discussions#4720] [Everyday usage]
### Output of `brew config`

```shell
brew config
HOMEBREW_VERSION: 4.1.5-7-gd30b687
ORIGIN: https://github.com/Homebrew/brew
HEAD: d30b68715fde642824f9e876b04102fb4f7dfcf6
Last commit: 9 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 15cc58f490ef218e460352ccd00ebf79f35b2491
Core tap last commit: 6 months ago
Core tap branch: master
Core tap JSON: 15 Aug 10:04 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.5-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
brew doctor
Your system is ready to brew.
```

### Description of issue

Hello,
I have a problem with filename sorting in MC. If sort-order is "Name" and "Case sensitive" disabled, files and directories with name starting c or C (small c or big C) are always first in the list (system locale en_US.UTF-8). Where to report this issue? Thanks.

**Comment by ZhongRuoyu** [comment]
According to their website (https://midnight-commander.org/#Issuetracking) issues should be reported to their issue tracker. See https://midnight-commander.org/wiki/NewTicket.

**Comment by mjosifek** [comment]
Why I'm asking here, I'm not able to reproduce this issue on Linux systems. This issue seems to be specific to macOS only. Or brew package?

**Reply by ZhongRuoyu** [reply]
I couldn't reproduce this issue on Linux and macOS (Intel) either. The behaviour was the same when I used the `mc` installed from Conda. There don't seem to be anything suspicious in the formula ([`midnight-commander.rb`](https://github.com/Homebrew/homebrew-core/blob/6199ea4d5090c54a9a64b1f50e32f6c0fbb3a65a/Formula/m/midnight-commander.rb)) so I suspect this is an upstream issue (if this wasn't with your configuration, etc.).
